resources/public/api/conf/2.0/testdata/fund-release-extension.md

```diff
@@ -533,6 +534,7 @@
 {
   "code": "FUND_RELEASE_SUPERSEDED",
   "message": "This fund release has already been superseded"
+  "lifeEventId": "6789000001"
```

Comment:
Please can you check and confirm this with Trevor? I would expect the lifeEventId returned for FUND_RELEASE_SUPERSEDED would be the ID for the updated fund release, not the ID for the extension?

Reply:
I have made the changes, can you please review. Thanks :)